tests/test_plotting.py

```diff
@@ -335,6 +335,29 @@ def test_IdealGas_plot_already_added():
         ValueError, match="Plot has already been added to this class instance"
     ):
         g.plot("v", "T")
+        
+def test_label_add_state():
+    """Test using a label in add_state."""
+    vd = VaporDome('water', ('v', 'T'))
+    st_1 = State('water', x = 1.0 * units.dimensionless, T= 100 * units.degC)
+    st_2 = State('water', x = 0.0 * units.dimensionless, T= 100 * units.degC)
+    assert(st_1._label is None)
+    assert(st_2._label is None)
+    vd.add_state(st_1, label = 1)
+    vd.add_state(st_2, label = "2")
+    assert(st_1._label == "1")
```

Comment:
In these two tests as well, don't use the `_label`, just use the regular `label` attribute.